Custom-Elements/elements.js

```diff
@@ -72,29 +91,22 @@ templateCountWords.innerHTML = `
   }
   </style>
   <div class="word-counter">
-    <textarea placeholder="write something here"></textarea>
+    <p>Acesta este un simplu text.</p>
     <button id="count-words-button">Count words</button>
```

Comment:
```suggestion
    <button id="reveal-count-button">Count words</button>
```

replace the id in all file

Reply:
the name of the custom element is self explanatory